modules/gitops-argocd-cli-utility-commands.adoc

```diff
@@ -0,0 +1,104 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/argocd-gitops-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="gitops-argocd-cli-utility-commands_{context}"]
+= Utility commands
+
+== argocd
+Parent command for {gitops-shortname} `argocd` CLI.
+
+.Example: Display all options
+[source,terminal]
+----
+$ argocd
+----
+
+== version
+Print version information of the CLI.
+
+.Command syntax
+`argocd version [flags]`
+
+.Example: Print the full version of client and server to stdout
+[source,terminal]
+----
+$ argocd version
+----
+
+.Example: Print only full version of the client, no connection to server will be made
+[source,terminal]
+----
+$ argocd version --client
+----
+
+.Example: Print only full version of the server
+[source,terminal]
+----
+$ argocd version --server openshift-gitops.openshift-gitops.crc.local
```

Comment:
I believe this should be replaced by the server address? 
cc: @anandf

Reply:
yes. will replace with a placedholder `<server_url>`